File: level1/javascript/행렬의 덧셈 12950.js
function solution(arr1, arr2) {
    // 방법 1
    var matrix = [];
    for(let i = 0; i < arr1.length; i++){
        var low = []
        for(let j = 0; j < arr1[0].length; j++){
            low.push(arr1[i][j] + arr2[i][j])
        }
        matrix.push(low);
    }
    return matrix;

    /* 방법 2
    return arr1.map((a,i)=>a.map((b,j) => b + arr2[i][j]));
    */ 
}

// 테스트 코드
console.log(solution([[1,2],[2,3]], [[3,4],[5,6]]));

File: level1/kotlin/행렬의 덧셈 12950.kt
class Solution {
    fun solution(arr1: Array<IntArray>, arr2: Array<IntArray>): Array<IntArray> {
        return arr1.mapIndexed { row, intArray ->
            intArray.mapIndexed { col, value ->
                value + arr2[row][col]
            }.toIntArray()
        }.toTypedArray()
    }
}